package.json

```diff
@@ -32,6 +32,7 @@
   "devDependencies": {
     "@types/jest": "^28.0.0",
     "@types/minimist": "^1.2.0",
+    "@types/readable-stream": "^2.3.14",
```

Comment:
I guess this one should be added as non-dev dep to `json-event-parser`.

Reply:
Not sure it's required. `json-event-parser` works from plain JS projects without needing `@types/readable-stream`.

Reply:
The convention in TS is that typings packages are included as non-dev deps, so that downstream users don't have to include all of these typing packages manually (which can incur in a lot of overhead for many transitive deps). And since typings packages are usually very small, this is not a problem for non-TS projects.

Reply:
Thank you! Done: https://github.com/comunica/json-event-parser.js